src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -113,7 +120,19 @@ private function execute(array $events, string $payload, Document $webhook, Docu
         }
 
         if (false === \curl_exec($ch)) {
-            $this->errors[] = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
+            $errorCount = $webhook->getAttribute('errors', 0) + 1;
```

Comment:
@vermakhushboo This could suffer from race conditions. We can use the increase method instead: 
https://github.com/utopia-php/database/blob/main/src/Database/Database.php#L3672